DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/ContentViewerViewModelTests.cs

```diff
@@ -19,32 +26,216 @@ public void SetUp()
+
+            // Then
+            contentViewerViewModel.CustomisationId.Should().Be(CustomisationId);
+        }
+
+        [Test]
+        public void Content_viewer_should_have_centreId()
+        {
+            // Given
+            var expectedTutorialContent = TutorialContentHelper.CreateDefaultTutorialContent();
+
+            // When
+            var contentViewerViewModel = new ContentViewerViewModel(
+                config,
+                expectedTutorialContent,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                TutorialId,
+                CandidateId,
+                ProgressId
+            );
+
+            // Then
+            contentViewerViewModel.CentreId.Should().Be(CentreId);
+        }
+
+        [Test]
+        public void Content_viewer_should_have_sectionId()
+        {
+            // Given
+            var expectedTutorialContent = TutorialContentHelper.CreateDefaultTutorialContent();
+
+            // When
+            var contentViewerViewModel = new ContentViewerViewModel(
+                config,
+                expectedTutorialContent,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                TutorialId,
+                CandidateId,
+                ProgressId
+            );
+
+            // Then
+            contentViewerViewModel.SectionId.Should().Be(SectionId);
+        }
+
+        [Test]
+        public void Content_viewer_should_have_tutorialId()
+        {
+            // Given
+            var expectedTutorialContent = TutorialContentHelper.CreateDefaultTutorialContent();
+
+            // When
+            var contentViewerViewModel = new ContentViewerViewModel(
+                config,
+                expectedTutorialContent,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                TutorialId,
+                CandidateId,
+                ProgressId
+            );
+
+            // Then
+            contentViewerViewModel.TutorialId.Should().Be(TutorialId);
+        }
+
+        [Test]
+        public void Content_viewer_should_have_progressId()
+        {
+            // Given
+            var expectedTutorialContent = TutorialContentHelper.CreateDefaultTutorialContent();
+
+            // When
+            var contentViewerViewModel = new ContentViewerViewModel(
+                config,
+                expectedTutorialContent,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                TutorialId,
+                CandidateId,
+                ProgressId
+            );
+
+            // Then
+            contentViewerViewModel.ProgressId.Should().Be(ProgressId);
+        }
+
+        [Test]
+        public void Content_viewer_should_have_tutorialName()
+        {
+            // Given
+            var expectedTutorialContent = TutorialContentHelper.CreateDefaultTutorialContent();

```

Comment:
I'd suggest setting the name when creating this, as it's important for the test 🙂

Reply:
Yep, done in 79d4b90 🙂